packages/tpetra/core/src/Tpetra_Details_packCrsMatrix_def.hpp

```diff
@@ -426,9 +426,9 @@ packCrsMatrixRow (const ColumnMap& col_map,
   const size_t num_ent_beg = offset;
   const size_t num_ent_len = PackTraits<LO, BDT>::packValueCount (num_ent_LO);
 
-  const GO gid = 0; // packValueCount wants this
+  const GO gid_example = 0; // packValueCount wants this
   const size_t gids_beg = num_ent_beg + num_ent_len;
-  const size_t gids_len = num_ent * PackTraits<GO, BDT>::packValueCount (gid);
+  const size_t gids_len = num_ent * PackTraits<GO, BDT>::packValueCount (gid_example);
```

Comment:
Since the input value of `packValueCount` doesn't matter for integers (like `GO`), we could just do this:
```
const size_t gids_len = num_ent * PackTraits<GO, BDT>::packValueCount (GO (0));
```
However, your fix is fine too :)